.htaccess

```diff
@@ -29,6 +29,10 @@ RewriteRule .* - [L]
 
 Redirect permanent /socs/ /groups
 
+# Block any dotfiles (and implicitly /.git, /.github, etcetera)
+RewriteCond %{REQUEST_URI} !^/?\.well-known/
```

Comment:
(Does the leading slash need to be optional here?)

Reply:
I remember some kind of difference surrounding the leading slash in 'server config' (i.e. apache2.conf) vs. in .htaccess context: i.e. in one it is present in the path, in the other it isn't -- this might be only for RewriteRule though, not REQUEST_URI.

Either way, I put the `/?` there because I couldn't be bothered to find out :P

Reply:
I have now bothered.  It's not optional.